package.json

```diff
@@ -197,6 +197,7 @@
     "p-map": "4.0.0",
     "p-queue": "6.6.2",
     "parse-link-header": "2.0.0",
+    "prettier": "2.7.1",
```

Comment:
Wouldn't this change affect the lock file?

Reply:
I'm not sure it distinguishes between dev and prod entries. And we should be running out CI with frozen lock file to detect if it's wrong

Reply:
it shouldn't change the lockfile